server/jetstream_test.go

```diff
@@ -17369,6 +17377,114 @@ func TestJetStreamWorkQueueSourceNamingRestart(t *testing.T) {
 	}
 }
 
+func TestJetStreamPullMaxBytes(t *testing.T) {
+	s := RunBasicJetStreamServer()
+	defer s.Shutdown()
```

Comment:
Move the defer shutdown past the defer removeDir please.

Reply:
ok done. I keep copying it from somewhere that is like that IIRC.